parm/templates/conTime.yaml

```diff
@@ -20,129 +20,129 @@ datasets:
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm492 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm498 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm504 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm510 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm516 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm522 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm528 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm534 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm540 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm546 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm552 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm558 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm564 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm570 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm576 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm582 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm588 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm594 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm600 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm606 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm612 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm618 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm624 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm630 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm636 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm642 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm648 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm654 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm660 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm666 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm672 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm678 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm684 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm690 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm696 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm702 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm708 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm714 | to_YMDH }}
-      - {{DATA}}/con_data/ges_{{OBSTYPE}}_stas.{{ PDATEm720 | to_YMDH }}
+      - {{DATA}}/ges_{{OBSTYPE}}_stas.{{ PDATE | to_YMDH }}
```

Comment:
Next time we tag up, remind me to talk about Jinja templating, we can write a Jinja template with a loop to simplify YAML like this